ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -408,7 +408,9 @@ trait UserManagementStorageBackend {
 
   def getUser(id: UserId)(connection: Connection): Option[UserManagementStorageBackend.DbUser]
 
-  def getUsers()(connection: Connection): Vector[User]
+  def getUsersOrderedById(after: UserId, maxResults: Int)(connection: Connection): Vector[User]
+
+  def getUsersOrderedById(maxResults: Int)(connection: Connection): Vector[User]
```

Comment:
```suggestion
  def getUsersOrderedById(fromExcl: Option[UserId], maxResults: Int)(connection: Connection): Vector[User]
```

Reply:
I wouldn't do that. See https://github.com/digital-asset/daml/pull/12610/files#r793848332

Reply:
For the record, applying suggested change